utility/src/main/java/org/oppia/android/util/logging/BUILD.bazel

```diff
@@ -0,0 +1,169 @@
+"""
+General purpose utilities for logging.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "annotations",
+    srcs = [
+        "LoggingAnnotations.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//third_party:javax_inject_javax_inject",
+    ],
+)
+
+kt_android_library(
+    name = "console_logger",
+    srcs = [
+        "ConsoleLogger.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        ":annotations",
+        ":log_level",
+        "//third_party:javax_inject_javax_inject",
+        "//third_party:org_jetbrains_kotlinx_kotlinx-coroutines-core",
+        "//utility/src/main/java/org/oppia/android/util/threading:annotations",
+    ],
+)
+
+kt_android_library(
+    name = "event_bundle_creator",
+    srcs = [
+        "EventBundleCreator.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:event_logger_java_proto_lite",
+    ],
+)
+
+kt_android_library(
+    name = "event_logger",
+    srcs = [
+        "EventLogger.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:event_logger_java_proto_lite",
+    ],
+)
+
+kt_android_library(
+    name = "exception_logger",
+    srcs = [
+        "ExceptionLogger.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+)
+
+kt_android_library(
```

Comment:
Nice that you added the firebase libraries as well. I created an additional task for that here:

https://github.com/oppia/oppia-android/issues/3076

Please assign yourself to that one as well. Could you move the firebase libraries to a separate BUILD.bazel file inside the firebase directory? That will make the distinction clearer.

Reply:
Ok @fsharpasharp I'll do it

Reply:
Done